src/components/Light/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import './style.css';
+
+import lightoff from './images/light-off.svg';
+import lighton from './images/light-on.svg';
+
+const Light = ({ name, state }) => {
+  const [light, setLight] = useState(state);
+
+  const switchLight = () => {
+    setLight(!light);
+  };
+  return (
+    <div className="light" onClick={switchLight}>
+      <div className="light__icon">
+        <img src={light ? lighton : lightoff} alt={name} />
```

Comment:
Když si zapneš aplikaci, tak vidíš že všechna světla svítí
![image](https://user-images.githubusercontent.com/8430123/141323614-6c805143-a0e1-436a-a311-98b7b8d342cd.png)
ale v datech je něco jiného:
![image](https://user-images.githubusercontent.com/8430123/141323556-aa3691aa-d616-4bd6-b65d-38d805d63569.png)
jak je to možné?

Reply:
Je to možné proto, že ty kontroluješ true/false, kdežto v datech chodí "on"/"off". Tj podmínka by měla být
`        <img src={light === "on" ? lighton : lightoff} alt={name} />`
(Je to proto, že se neprázdný string je vždycky truthy, kdežto prázdný falsy. Tj "off" je truthy. https://www.sitepoint.com/javascript-truthy-falsy/)

Reply:
dakujem, toto si musim este viac pozriet